test/fw/bin/pbs_snapshot

```diff
@@ -305,10 +325,28 @@ if __name__ == '__main__':
 
         # Delete the snapshot directory itself
         du.rm(path=main_snap, recursive=True, force=True)
+    elif not du.is_localhost(primary_host):
+        # Capture snapshot of a remote host
+
+        thread = Thread(target=remotesnap_thread, args=(
+            ptl_logger, primary_host))
+        thread.start()
+        thread.join()
+        p_snappath = os.path.join(out_dir, primary_host + "_snapshot.tgz")
+        tar = tarfile.open(p_snappath)
+        main_snap = tar.getnames()[0].split(os.sep, 1)[0]
+        tar.close()
+        main_snap = os.path.join(os.path.abspath(out_dir), main_snap)
```

Comment:
Not that it really matters, but if you want to be consistent with what you did above, swap the order of these two lines.